test/test_util.h

```diff
@@ -631,6 +651,7 @@ __host__ __device__ __forceinline__ void InitValue(GenMode /* gen_mode */,
 /**
  * cub::KeyValuePair<OffsetT, ValueT>test initialization
  */
+#pragma nv_exec_check_disable
```

Comment:
I agree, this is hacky and not-great. It definitely makes sense to refactor this as we improve CUB's tests. Maybe something like thrust_benchmark's `range_generator` could be used instead?

But since it's just a testing facility with no user exposure, this is very low priority. We can look into this while doing test clean-ups, but for now let's just get things building again to unblock gpuCI 👍